gomint-server/src/main/java/io/gomint/server/world/block/Leaves.java

```diff
@@ -66,6 +70,17 @@ public BlockType getType() {
         return BlockType.LEAVES;
     }
 
+    @Override
+    public List<ItemStack> getDrops( ItemStack itemInHand ) {
+        if ( isCorrectTool( itemInHand ) ) {
```

Comment:
Please always init a List before checking isCorrectTool and fill in the items if needed